mino/minogrpc/overlay.proto

```diff
@@ -4,6 +4,29 @@ package minogrpc;
 
 import "google/protobuf/any.proto";
 
+// Certificate is wrapper around a x509 raw certificate and its address.
+message Certificate {
+    bytes address = 1;
+    bytes value = 2;
+}
+
+message CertificateAck {}
+
+// JoinRequest sends a request to join a network to a distant node. It must
+// contain a valid token and its own certificate.
+message JoinRequest {
+    string token = 1;
```

Comment:
Is the token a hex encoded string or really any unicode char?

Reply:
It's a string. That's left to the implementation to generate bytes and convert them in hex (or base64 like it's doing now)